api/routes/dto.py

```diff
@@ -0,0 +1,8 @@
+from pydantic import BaseModel
+from typing import Optional, Union
+
+class UniversalResponseDTO(BaseModel):
```

Comment:
MINOR: 
It's alright, but "Universal" in the name always triggers me. Maybe we can rename it somehow